acmw-db-app/components/GHCVolunteerForm.jsx

```diff
@@ -0,0 +1,65 @@
+import SubmitButton from './FormComponents/SubmitButton'
+import TextField from './FormComponents/TextField'
+import SelectInput from './FormComponents/SelectInput'
+import MultiSelectInput from './FormComponents/MultiSelectInput'
+import styles from '../styles/components/CompanyForm.module.css'
+import {useState} from 'react'
+
+const GHCVolunteerForm = (props) => {
+
+    // eventually connect this to actual GHC scholarship awardees from database
+    // Can implement a search similar to "company search component" but for members
+    const GHCNames = ["Milly Mason", "Jing George", "Angela Li", "Morgan Zahner", "Sara Miskus", "Amy Huang"]
+
+    const [GHCName, setGHCName] = useState("")
+    const [volunteerEventName, setVolunteerEventName] = useState("")
+    const [numHours, setNumHours] = useState(0)
+    const [hourError, setHourError] = useState("")
+    const [eventError, setEventError] = useState("")
+
+    const onSubmit = () => {
+        if (volunteerEventName == "" || numHours == 0) {
+            setEventError("Event name must not be blank!")
+            setHourError("Must provide a number greater than 0!")
```

Comment:
I think I'd prefer it if the error messages could show up for just one (depending on which is wrong) instead of both every time.